docs/content/guides/6.multistore/3.patterns/2.design/4.different-code-per-store.md

```diff
@@ -1,8 +1,208 @@
 ---
-title: Different markups for individual components
+title: Different Markups for Individual Components
 layout: default
 navigation:
   icon: tabler:number-1-small
 ---
 
-# Different markups for individual components
+# Different Markups for Individual Components
+
+> **Prerequisite:** Familiarity with [Tooling & Concepts → File-Based Inheritance](../2.file-based-inheritance.md) is required to understand file override mechanisms and inheritance principles.
+
+**What You’ll Learn**
+
+::list{type="success"}
+- How to safely modify the markup of individual components in a multibrand project without breaking their props interfaces
+- How to handle breaking changes in component props with step-by-step workflows, including duplicating and modifying files
+- Best practices for managing component markup customization to maintain consistency and avoid duplication
+- How to implement these changes using examples in Next.js and Nuxt to adapt to specific frameworks
+::
+
+## Why Customize Component Markups?
+
+In multibrand setups, different stores often require unique layouts or styling. Customization is necessary to ensure proper alignment with branding and design guidelines while sharing common functionalities across stores. For example:
+
+- Store-specific customizations to match unique branding designs.
+- Adjustments to the structure or layout of a component in one brand without affecting others.
+
+Changing a component’s markup is a **structured process**, and this guide will walk you through **safe modifications** and scenarios where **breaking changes** are unavoidable.
+
+## Safe Markup Changes Without Breaking Props
+
+The most straightforward way to customize a component is by modifying **only its markup** while keeping its props interface unchanged. This ensures compatibility across the codebase and minimizes any disruption to dependent components or features.
+
+### When to Use
+
+- To rearrange the component structure.
+- To adjust styling for specific stores or brands without altering the functionality.
+
+### The Solution: Component Override
+
+In cases where a specific store requires customization, you can override the component by duplicating and modifying it within that store's directory.
+
+#### Example Workflow
+
+1. **Duplicate** the component inside the store-specific directory.
+2. **Modify** the duplicated component as per the store's requirements (e.g., layout, structure, or styles).
+
+#### In Next.js
+
+Here’s how to override `ExampleComponent` for a specific store (`store-a`) in a Next.js project:
+
+```tsx
+// /apps/stores/store-a/storefront-unified-nextjs/components/example-component.tsx
+
+interface ExampleComponentProps {
+  title: string;
+  imageSrc: string;
+}
+
+export default function ExampleComponent({ title, imageSrc }: ExampleComponentProps) {
+  return (
+    <div className="store-a example-component"> {/* Customized className for store-a */}
```

Comment:
we use tailwindcss, so it would be better to also use it in the example component

Reply:
Changed